MAUI/Scheduler/month-view.md

```diff
@@ -79,6 +79,32 @@ this.Scheduler.MonthView.ShowLeadingAndTrailingDates = false;
 
 ![show-or-hide-leading-and-trailing-days-appearence-customization-in-maui-scheduler](images/month-view/show-or-hide-leading-and-trailing-days-appearence-customization-in-maui-scheduler.png)
 
+## Customize number of month rows
+
+The number of weeks in the month view can be changed by setting the [NumberOfVisibleWeeks]() property in the [MonthView](). By default, [NumberOfVisibleWeeks]() starts from current week, and this can be modified using the initialDisplayDate property of scheduler, the two weeks scheduler can be achieved by setting the `NumberOfVisibleWeeks` property with the value 2.
```

Comment:
Remove this and add  API remark in Note

By default, [NumberOfVisibleWeeks]() starts from current week, and this can be modified using the initialDisplayDate property of scheduler, the two weeks scheduler can be achieved by setting the `NumberOfVisibleWeeks` property with the value 2.